.github/workflows/main.yml

```diff
@@ -16,34 +16,45 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
         with:
-          python-version: "3.8"
-      - uses: eifinger/setup-rye@v2.6
-      - run: rye config --set-bool behavior.use-uv=true
-      - run: rye sync --no-lock
+          version: "0.4.12"
```

Comment:
Can we just use `latest`?

Reply:
We can. I don't want to not pin the dependencies.